tests/kubernetes-agent/KubernetesAgent.IntegrationTests/Setup/Common/TestLogger.cs

```diff
@@ -1,225 +1,37 @@
 using System;
-using Octopus.Diagnostics;
-using ILog = Octopus.Diagnostics.ILog;
+using Octopus.Tentacle.Contracts.Logging;
 
 namespace KubernetesAgent.Integration.Setup.Common
 {
-    public sealed class TestLogger(ILogger logger) : ILog, IDisposable
```

Comment:
`Tentacle.Client` no longer references `Octopus.Diagnostics` (which has been archived) so we have to use the new `Tentacle.Client` logger interface `ITentacleClientTaskLog`.